src/components/Button.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+function Button(props) {
+    return (
+        <div>
+            <button type="button" className="btn"
+                    onClick={Clickhandler}
```

Comment:
Ook functienamen camelCase schrijven
```suggestion
                    onClick={clickHandler}
```